src/main/java/org/fundacionjala/at13/spaceinvaders/GameBoard.java

```diff
@@ -25,6 +40,8 @@ public GameBoard() {
      */
     public void init() {
 
+        timer = new Timer();
+
```

Comment:
maybe remove the spaces between variables

Reply:
okay 👍🏻

Reply:
Already solved 😁

Reply:
great dani